internal/service/adapters.go

```diff
@@ -222,7 +225,16 @@ func createRedisClient(c config.Config, url string) (*redis.Client, error) {
 		return nil, fmt.Errorf("invalid redis URL: %w", err)
 	}
 	opts.PoolSize = c.GetInt(redisPoolSizePath)
-	return redis.NewClient(opts), nil
+
+	hostPort := strings.Split(opts.Addr, ":")
+
+	client := redis.NewClient(opts)
+	client.AddHook(redisotel.NewTracingHook(redisotel.WithAttributes(
+		semconv.NetPeerNameKey.String(hostPort[0]),
+		semconv.NetPeerPortKey.String(hostPort[1])),
+	))
```

Comment:
Should it not check if `tracing` is enabled to add this hook?

Reply:
Ops, I'll add the check!